src/main/java/com/google/sps/data/UserRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+import java.util.Collection;
+
+/** interface for reading/writing user data that is abstracted from the database used. */
+public interface UserRepository {
+
+  public void addUser(User user);
+
+  public void removeUser(User user);
+
+  public Collection<User> getUsers();
```

Comment:
I imagine you might add additional getUser methods here (e.g. getUserById, getUserByName, etc). Maybe name this getAllUsers for clarity.

Reply:
Good idea, done